### PR TITLE
removed reference to whitesource-ft for xmodule

### PR DESCRIPTION
**Pull request by samq-ws**

### CODE REVIEW

File: Prioritize/DotNet/Multi-Module/xModuleAnalyzer-NET.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+do
+# For each dll in the above list, print out the variables that will be used for the prioritize scan
+echo "appPath:" $DLL
+DIR="$(echo "$DLL" | awk -F "$RELEASEDIR" '{print $1}')"
+echo "directory:" $DIR
+PROJECT=$(basename $DLL .dll)
+echo "PROJECT:" $PROJECT
+# Run a WSS prioritize scan for each DLL in the multi-module.txt file
+java -jar wss-unified-agent.jar -appPath "$DLL" -d "$DIR" -project "$PROJECT"
+done
```
Comment (tidharm) [inline Prioritize/DotNet/Multi-Module/xModuleAnalyzer-NET.sh:31]: Missing newline

File: Prioritize/DotNet/Multi-Module/github-action_linux.yml [hunk 2]
```diff
@@ -32,10 +32,12 @@ jobs:
...
         WS_EXCLUDES: '**/build/** **/tests/**'
+        SEARCHDIR: $(GITHUB_WORKSPACE)
+        RELEASEDIR: /bin/
       run: |
         curl -LJO https://unified-agent.s3.amazonaws.com/wss-unified-agent.jar
         echo Unified Agent downloaded successfully
-        curl -LJO https://raw.githubusercontent.com/whitesource-ft/xModuleAnalyzer-NET/main/xModuleAnalyzer-NET.sh
+        curl -LJO https://raw.githubusercontent.com/mend-toolkit/mend-examples/Prioritize/DotNet/Multi-Module/xModuleAnalyzer-NET.sh
         echo xModuleAnalyzer-NET downloaded successfully
         chmod +x ./xModuleAnalyzer-NET.sh && ./xModuleAnalyzer-NET.sh
```
Comment (tidharm) [inline Prioritize/DotNet/Multi-Module/github-action_linux.yml:43]: Missing newline

**Review by tidharm — CHANGES_REQUESTED** [review]

File: Prioritize/DotNet/Multi-Module/azure-pipelines_linux.yml [hunk 3]
```diff
@@ -16,7 +16,7 @@ steps:
 - script: |
     curl -LJO https://unified-agent.s3.amazonaws.com/wss-unified-agent.jar
     echo Unified Agent downloaded successfully
-    curl -LJO https://raw.githubusercontent.com/whitesource-ft/xModuleAnalyzer-NET/main/xModuleAnalyzer-NET.sh
+    curl -LJO https://raw.githubusercontent.com/mend-toolkit/mend-examples/Prioritize/DotNet/Multi-Module/xModuleAnalyzer-NET.sh
```
[the remark this answers is not in the document]
Comment (samq-ws) [reply Prioritize/DotNet/Multi-Module/azure-pipelines_linux.yml:19]: This URL wont go anywhere until you merge this PR ;-)

File: Prioritize/DotNet/Multi-Module/github-action_linux.yml [hunk 4]
```diff
@@ -32,10 +32,12 @@ jobs:
         WS_FILESYSTEMSCAN: false
         WS_GENERATEPROJECTDETAILSJSON: true
         WS_EXCLUDES: '**/build/** **/tests/**'
+        SEARCHDIR: $(GITHUB_WORKSPACE)
+        RELEASEDIR: /bin/
       run: |
         curl -LJO https://unified-agent.s3.amazonaws.com/wss-unified-agent.jar
         echo Unified Agent downloaded successfully
-        curl -LJO https://raw.githubusercontent.com/whitesource-ft/xModuleAnalyzer-NET/main/xModuleAnalyzer-NET.sh
+        curl -LJO https://raw.githubusercontent.com/mend-toolkit/mend-examples/Prioritize/DotNet/Multi-Module/xModuleAnalyzer-NET.sh
```
[the remark this answers is not in the document]
Comment (samq-ws) [reply Prioritize/DotNet/Multi-Module/github-action_linux.yml:40]: This URL wont go anywhere until you merge this PR ;-)

File: Prioritize/README.md [hunk 5]
```diff
@@ -15,7 +15,7 @@ For all examples above, make sure to change the branches defined within the .yml
 
 **Important .NET Note** 
 <br>
-[xModuleAnalyzer](https://github.com/whitesource-ft/xModuleAnalyzer-NET) scripts may require some customization due to different build and exclusion types
+[xModuleAnalyzer](https://github.com/mend-toolkit/mend-examples/Prioritize/DotNet/Multi-Module/xModuleAnalyzer-NET.sh) script may require some customization due to different build and exclusion types
```
[the remark this answers is not in the document]
Comment (samq-ws) [reply Prioritize/README.md:18]: This URL wont go anywhere until you merge this PR ;-)

**Review by clindseyws — DISMISSED** [review]

**Review by tidharm — APPROVED** [review]

**Pull request merged**